scripts/gcp_setup/setup_forseti.py

```diff
@@ -40,6 +40,15 @@ def run():
     group.add_argument('--cloudsql-region',
                        help='The Cloud SQL region')
 
+    network = parser.add_argument_group(title='network')
+    network.add_argument('--network-host-project-id',
```

Comment:
I know this is a port from the previous PR, but I am wondering if we can take the chance to improve the naming?
`--vpc-host-project-id` ?